contentcuration/contentcuration/utils/nodes.py

```diff
@@ -157,3 +162,119 @@ def filter_out_nones(data):
     Filter out any falsey values from data.
     """
     return (d for d in data if d)
+
+
+def _get_diff_filepath(node_id1, node_id2):
+    return os.path.join(settings.DIFFS_ROOT, node_id1, '{}.json'.format(node_id2))
+
+
+def _get_created_time(node):
+    return node.created.strftime('%Y-%m-%d %H:%M:%S')
+
+
+def get_diff(updated, original):
+    jsonpath = _get_diff_filepath(updated.pk, original.pk)
+    if default_storage.exists(jsonpath):
+        with default_storage.open(jsonpath, 'rb') as jsonfile:
+            data = json.load(jsonfile)
+            if data['generated'] == _get_created_time(updated):
+                return data
+    return None
+
+
+def generate_diff(updated_id, original_id):
+    import time
+    time.sleep(20)
```

Comment:
What is this sleep call for?

Reply:
oh oops! Must have been left over from when I was testing longer calls. Removing now